src/core.js

```diff
@@ -2158,10 +2168,15 @@ imgix.fluid = function(elem) {
 	var fluidElements;
 	if (elem && !imgix.helpers.isFluidSet(elem)) {
 		fluidElements = Array.isArray(elem) ? elem : [elem];
-	} else {
+  } else {
 		var cls = '' + options.fluidClass;
 		cls = cls.slice(0, 1) === '.' ? cls : ('.' + cls);
-		fluidElements = document.querySelectorAll(cls);
+    if (node && imgix.isImageElement(node)) {
```

Comment:
I was attempting to guard against passing a leaf node directly to `fluid`, since `querySelectorAll` won't include the element it's called on. Maybe a better alternative, and one that eliminates the edge case, is to do something like:
```js
fluidElements = rootNode.querySelectorAll(cls);
fluidElements.push(rootNode);
```
(I'm pretty sure `querySelectorAll` doesn't return a proper Array and we'd have to do some slicing here, but that's the gist.)